content/docs/sidebar.json

```diff
@@ -552,46 +552,27 @@
     ]
   },
   {
-    "label": "Studio",
     "slug": "studio",
-    "source": "studio/index.md",
     "icon": "studio",
     "children": [
-      {
-        "label": "How It Works",
-        "slug": "overview"
-      },
+      "how-it-works",
       "get-started",
       {
-        "label": "User Guide",
         "slug": "user-guide",
-        "source": "user-guide/index.md",
+        "source": false,
```

Comment:
- Deleted the [index page](https://dvc.org/doc/studio/user-guide) for the Studio UG, which doesn't seem to provide much value IMO (but it can be easily get restored if you prefer).

Reply:
👍